.gitignore

```diff
@@ -0,0 +1,3 @@
+
+./Welolo/node_modules
+./Welolo/client/node_modules
```

Comment:
Looks good to me! I'm not sure if you wanted to delete the client/node_modules folder with this PR, but it doesn't look like it got deleted like the Welolo/node_modules folder did.

Reply:
I'm not seeing client/node_modules when I pull to my local repo. Are you still seeing it?